CMakeLists.txt

```diff
@@ -172,6 +172,20 @@ set(WARN_FLAGS ${WARN_FLAGS} $<$<COMPILE_LANGUAGE:CXX>:-Werror>)
 set (CUTLASS_INC "")
 target_compile_definitions(matx INTERFACE MATX_ENABLE_CUTLASS=0)
 
+if (MATX_NVTX_FLAGS)
+    add_definitions(-DMATX_NVTX_FLAGS)
+    target_compile_definitions(matx INTERFACE MATX_NVTX_FLAGS)
+endif()
+if (MATX_BUILD_32_BIT)
+    set(INT_TYPE "lp64")
```

Comment:
Should this be called `BLAS_INT_TYPE`?

Reply:
Other NVPL libraries like LAPACK also follow this convention of 2 interfaces. ArmPL also uses this convention. Should we use some other name?

Reply:
Nope, this is fine.